src/utils/ProtectedRoute.tsx

```diff
@@ -0,0 +1,19 @@
+
+import React from 'react'
+
+import { Navigate } from 'react-router-dom'
+
+import LoginService from '../services/LoginService'
+
+
+const ProtectedRoute = ({ children }: {children: JSX.Element} ) => {
+
+  !LoginService.isLoggedIn()
```

Comment:
What is the purpose of `!LoginService.isLoggedIn()` on line number 11?

Reply:
Purpose of this line was testing. I was forrgoten to remove it. I will remove it.